templates/src/components/PrefetchLink/PrefetchLink.js

```diff
@@ -4,19 +4,22 @@ import classnames from 'classnames';
 import { BaseLink } from '@jam3/react-ui';
 import prefetch from 'inject-prefetch';
 import checkProps from '@jam3/react-check-extra-props';
-import fileExtension from 'file-extension';
 
 import routeKeys from '../../routes/keys';
 
 import './PrefetchLink.scss';
 
 const PrefetchLink = React.memo(
   React.forwardRef((props, ref) => {
+    function getFileExtension2(filename) {
```

Comment:
Can we remove the number `2` from the function name?
```suggestion
    function getFileExtension(filename) {
```